roles/foreman_installer/tasks/upgrade.yml

```diff
@@ -7,6 +7,8 @@
   import_role:
     name: update_os_packages
 
+- include_tasks: installer_version.yml
```

Comment:
interestingly, I think the "add `--upgrade`" logic in `upgrade.yml` was broken before this, as the version was read before we upgraded the packages

Reply:
ah, it was not, as we upgrade the installer package before calling `upgrade.yml`, clever.

Reply:
I'd like to kill this task eventually. Does that mean that `update_os_packages` is redundant and can be dropped?

Reply:
No, it does not mean that. Unless you want a machine with upgraded installer, but not upgraded everything else ;)